sdk/communication/Azure.Communication.Common/src/AutoRefreshTokenCredential.cs

```diff
@@ -6,6 +6,8 @@
 using System.Threading.Tasks;
 using Azure.Core;
 
+#nullable enable
```

Comment:
`AutoRefreshTokenCredential` is an internal type, so we do not have to disable nullable ref-types